src/sdk/namespace_fs.js

```diff
@@ -1011,6 +1042,30 @@ class NamespaceFS {
         return p.endsWith('/') ? p + config.NSFS_FOLDER_OBJECT_NAME : p;
     }
 
+    _assign_md5_to_fs_xattr(md5_digest, fs_xattr) {
+        // TODO: Assign content_md5_mtime
+        fs_xattr = Object.assign(fs_xattr || {}, {
+            [XATTR_MD5_KEY]: md5_digest
+        });
+        return fs_xattr;
```

Comment:
Why not initialize the object from the caller, and then just update the fs_xattr object?
```
fs_xattr[XATTR_MD5_KEY] = md5_digest;
```